### ISSUE TITLE
New activity type: Announcements

### ISSUE THREAD

**Issue by AniOnGit** [wwevo/guildhome#63]
I thought about how people can better notice about new features like the timezone stuff. Then I came to a new activity type announcements. Like the ActionMessages they should look different, having no title and no user info in them. They need a status - active/inactive - which when set to active makes them be on the very top of the activity stream, even above featured events, until someone marks them inactive. 

I don't know if there is already user rights management implemented, but not everyone should be able to set up announcements. I'd say only leaders, maaaayve officers, but not more.

**Comment by wwevo** [comment]
No User-rights management so far. There's basic user-gating for Admin, Operator and User, those need to be hard-coded atm. That can be extended in the future though.

Can't really wrap my head around this one. Can we get a bit more technical? Every user needs to check them as read? what happens when one didn't log in for two months and gets 100 of them? Where do they go when they are not active? Still in the stream but in another location? Will there be announcements popping up in the middle of the stream after they get marked read?

Sorry, it's Pre-Coffee-Time ^^

**Comment by AniOnGit** [comment]
Yeah, it's totally without coffee time here ;) I didn't meant the user check them as read, the admin or poster deactivates them when he/she thinks the announcement got to enough brains to be known ;)

For the technical side, it's basically a new activity type with a new table containing just the message and an active flag. Only active announcements will be shown. I have not thought further into this, but I guess every other functionality like seeing deactivated ones and re-activate them might a) not necessary or b) come later ;)
